internal/model/responses.go

```diff
@@ -0,0 +1,5 @@
+package model
+
+type ResponseHttpError struct {
```

Comment:
This can be moved to `internal/http/response/error.go` and it can be called `ErrorResponse`.